components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/resources/identity-governance-3.yaml

```diff
@@ -0,0 +1,437 @@
+    PropertyRes:
+      type: object
+      description: Governance connector property.
+      properties:
+        name:
+          type: string
+          description: Property name.
+          example: suspension.notification.enable
+        value:
+          type: string
+          description: Property value.
+          example: 'false'
+        displayName:
+          type: string
+          description: Property display name.
+          example: Enable
+        description:
+          type: string
+          description: Property description.
+          example: Enable account suspend notifications.
+    ConnectorsPatchReq:
+      type: object
+      description: Governance connector property patch request.
+      required:
+        - operation
+        - properties
+      properties:
+        operation:
+          type: string
+          description: Governance connector properties patch operation.
+          enum:
+            - UPDATE
+          example: UPDATE
+        properties:
+          type: array
+          description: Governance connector properties to patch.
+          items:
+            $ref: '#/components/schemas/PropertyReq'
+    PropertyReq:
+      type: object
+      description: Governance connector property to patch.
+      required:
+        - name
+        - value
+      properties:
+        name:
+          type: string
+          description: Property name.
+          example: suspension.notification.enable
+        value:
+          type: string
+          description: Property value.
+          example: 'false'
+    Link:
+      type: object
+      properties:
+        href:
+          type: string
+          description: Path to the target resource.
+        rel:
+          type: string
+          enum:
+            - CATEGORY
+            - CONNECTOR
+          description: Describes how the current context is related to the target resource.
+    CategoryLink:
+      allOf:
+        - $ref: '#/components/schemas/Link'
+      properties:
+        rel:
+          example:
+            - CATEGORY
+        href:
+          example: >-
+            /t/carbon.super/api/server/v1/identity-governance/QWNjb3VudCBNYW5hZ2VtZW50IFBvbGljaWVz
+    ConnectorLink:
+      allOf:
+        - $ref: '#/components/schemas/Link'
+      properties:
+        rel:
+          example:
+            - CONNECTOR
+        href:
+          example: >-
+            /t/carbon.super/api/server/v1/identity-governance/QWNjb3VudCBNYW5hZ2VtZW50IFBvbGljaWVz/connectors/c3VzcGVuc2lvbi5ub3RpZmljYXRpb24
+    Error:
+      type: object
+      required:
+        - code
+        - message
+      properties:
+        code:
+          type: string
+          example: AAA-00000
+        message:
+          type: string
+          example: Some Error Message
+        description:
+          type: string
+          example: Some Error Description
+        traceId:
+          type: string
+          example: e0fbcfeb-3617-43c4-8dd0-7b7d38e13047
```

Comment:
One more thing, we need to have the host and base path correctly to write integration tests. So better add that as well before merging

Reply:
I don't understand what you mean. Can you please explain? The place you are pointing to does not have any "host and base path"